src/cmd/completions.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var completionsNoDesc bool
+
+var completionsCmd = &cobra.Command{
+	Use:   "completions [bash|zsh|fish|powershell]",
+	Short: "Generates auto-completion scripts for Bash, zsh, fish, and PowerShell",
+	Long: `Generates auto-completion scripts for Bash, zsh, fish, and PowerShell
+
+With completions enabled, git-town TAB will show you all possible
+subcommands. As a bonus, some shells even show the short help text next to it.
+
+To enable completions:
+
+Bash:
+
+$ source <(git-town completions bash)
+
+Persist and autoload on each session:
+
+Linux: $ git-town completions bash > /etc/bash_completion.d/git-town
+
+MacOS: $ git-town completions bash > /usr/local/etc/bash_completion.d/git-town
+
+Zsh:
+
+$ source <(git-town completions zsh)
+
+Persist and autoload on each session:
+
+$ git-town completions zsh > /usr/share/zsh/vendor-completions/_git-town
```

Comment:
That would be suicide... :D
zsh has a bunch of places where to put those.
For oh-my-zsh users it could be: `~/.oh-my-zsh/completions` - but then, if you're not a oh-my-zsh user...

`vendor` is just the place for vendors (if we where to package git-town with the auto-packager tool, this is where it should go)

Reply:
Okay, I just installed ZSH using HomeBrew for Linux, and it still has `/usr/share/zsh/vendor-completions`. I'm fine going with that.  🙂

Reply:
Nice, thanks you double checked!